src/service/resource.js

```diff
@@ -441,12 +402,7 @@ export class Resource {
    */
   completeCollapse() {
     toggle(this.element, false);
-    this.layoutBox_ = layoutRectLtwh(
-        this.layoutBox_.left,
-        this.layoutBox_.top,
-        0, 0);
-    this.isFixed_ = false;
-    this.element.updateLayoutBox(this.layoutBox_);
+    this.requestMeasure();
```

Comment:
The layout box will be updated on the next pass, but `requestMeasure` doesn't schedule a resource pass. With this change, we should make sure this method is only called when a resource pass is scheduled.